extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -31,14 +31,24 @@ public function getSource() {
 	}
 
 	public function getSourceLabel() {
-		$source = $this->extractSourceFromNode( $this->xmlNode )[0];
+		$sourceLabels = [];
+		$sources = $this->extractSourceFromNode( $this->xmlNode );
 		$label = \Sanitizer::stripAllTags( $this->getInnerValue( $this->xmlNode->{self::LABEL_TAG_NAME} ) );
 
-		if ( !empty( $source ) ) {
-			return [ $source => $label ];
+		if ( count( $sources ) > 1 ) {
+			foreach ( $sources as $source ) {
+				if ( !empty( $source ) ) {
+					$sourceLabel = !empty( $label ) ? "{$label} ({$source})" : $label;
+					$sourceLabels = array_merge( $sourceLabels, [ $source => $sourceLabel ] );
```

Comment:
Is there any reason for ``array_merge`` here? Why not ``$sourceLabels[$source] = $sourceLablel``?